src/style.css

```
@@ -0,0 +1,79 @@
+@font-face {
+  font-family: "Avenir";
+  src: url(fonts/Avenir-Book.ttf);
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: "Avenir_bold";
+  src: url(fonts/Avenir-Heavy.ttf);
+  font-weight: 400;
+}
+
+html {
+  --font-color: #3d4e61;
+  --background-color: #fff;
+  --big-search-bar: 23px;
+  --small-search-bar: 10px;
+  --border-color: #e1e7ed;
+  --box-shadow-color: rgba(61, 78, 97, 0.1);
+  --linear-gradient-settings: 180deg, #fff 0%, #f6f6f7 100%;
+
+  color: var(--font-color);
+}
+
+.search_container {
+  margin-top: 20px;
+  position: relative;
+}
+
+.search_icon--1 {
+  position: absolute;
+  top: 25px;
+  left: 26px;
+}
+
+.search_icon--2 {
+  position: absolute;
+  height: 11px;
+  top: 15px;
+  left: 13px;
+}
```

after fix classes in html
```suggestion
.search_icon {
  position: absolute;
  top: 25px;
  left: 26px;
}

.search_icon--small {
  height: 11px;
  top: 15px;
  left: 13px;
}
```